code/modules/recycling/disposal.dm

```diff
@@ -1355,7 +1355,7 @@
 	// Unlink trunk and disposal so that objets are not sent to nullspace
 	if (linked)
 		var/obj/machinery/disposal/D = linked
-		D.trunk = null
+		D?.trunk = null
```

Comment:
Can a type check work here, not sure this will fix?

Reply:
yeah changed it to istype